tests/unit/NameRequestInfo.spec.ts

```diff
@@ -94,7 +98,7 @@ describe('Name Request Info with a NR', () => {
   })
 
   it('renders the Name Request applicant information with data', async () => {
-    await wrapper.vm.$store.commit('mutateNameRequest', { ...mockNrData })
+    store.stateModel.nameRequest = mockNrData
```

Comment:
Since you're not awaiting a commit here, do you need to await Vue.nextTick() ?

Reply:
Test fails if I do not add that but I will try to find a solution for it and see if we can remove it.

Reply:
@ketaki-deodhar @JazzarKarim One thing you guys may have picked up is that our unit tests sometimes test for the absence of something, but it's actually absent because the unit test hasn't set up something critical. So the test passes but the component isn't actually being tested.

The solution to this is better unit tests. Just something to keep in mind. I often look at `wrapper.html()` to see what the unit test is actually rendered and whether it matches my expectations.

Reply:
_Maybe_ you can await the setter (if there is one).

I'm OK with awaiting for the next timeslice (next tick) in limited situations, like this one.